tests/misc/memestra_nb_demo.ipynb

```diff
@@ -2,18 +2,9 @@
  "cells": [
```

Comment:
Why are these changes needed?

Reply:
I wanted to have a deprecated function call on a column other than 0, to check that also. I cleared all the outputs but you're right, it can be interesting to check the mix of different outputs, from the deprecation function and the preprocessor.